### PR TITLE
New Stealth Traitor Item: Armor Polish

### PR DESCRIPTION
**Pull request by ToasterBiome**
# General Documentation

image

**NEW:**

image

Armor Polish: Increases the armor of whatever suit/hat slot item you apply it to be 30 melee, 30 bullet, 20 laser, 25 energy. This does not change bomb/bio/rad/fire/acid resistance on it, so it will stay the same as the armor values before it. It wont let you apply it to things that are higher than these values, lest you waste your points, and if your armor values are higher than what they change to they will stay the same. 6 TC. This makes it roughly on par with an armor vest (30/30/30/10) while looking cool for fashionstation. Two uses (for suit/hat)

### Intent of your Pull Request

Discord_TF5Xe5VuSs
Discord_ushMi1JaQu

### Why is this change good for the game?

As a traitor I often find myself wanting to be more "drippy", as the kids call it. Stealing an armor vest has it's charms or wearing a clunky suit, but I wanna look cool when I get validhunted.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
New traitor item! It will need an entry on the Syndicate Items page. It's a stealthy item.

### What should players be aware of when it comes to the changes your PR is implementing?

Now any graytider can be armored up and you'll never know!

### What general grouping does this PR fall under? 
Traitor Items

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
No!
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
30 melee/30 bullet/20 laser/25 energy

# Changelog

:cl:  
rscadd: Added armor polish
rscadd: Added armor polish being purchasable from the traitor uplink
/:cl:

I'd like some feedback on sprite/armor values/tc cost anything really

### CODE REVIEW

**Comment by LazennG** [comment]
drip good yes i wanna see mroe people style

**Comment by Jarod1200** [comment]
Drip

**Comment by ChesterTheCheesy** [comment]
Cool beans although the sprite for item looks terrible

**Review by nesisknesis — CHANGES_REQUESTED** [review]
You're deleting sprites from a recently merged PR.
image

**Comment by ToasterBiome** [comment]
> You're deleting sprites from a recently merged PR.
> image

Ahh I see, the branch was created before it was merged, I’ll fix it right up

**Comment by ToasterBiome** [comment]
new icon 
image

**Comment by UselessTheremin** [comment]
fix yer conflicts nerd

**Comment by ToasterBiome** [comment]
> fix yer conflicts nerd

I don’t know why map diff is shitting itself cause this PR doesn’t touch maps

File: code/modules/uplink/uplink_items.dm
Comment (Hopekz) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
	desc = "This two use polish will strengthen clothing to be as strong as a standard-issue armor vest.  \
			Reinforced with nanite technology, you are able to stay looking good while bashing heads in. \
			Beware, you can only polish things that go over your jumpsuit or a hat!"
```

**Review by nesisknesis — APPROVED** [review]

**Comment by SomeguyManperson** [comment]
pull from upstream master, go through whatever conflict resolution you have to, make sure no unrelated images are changed/all related images are present then push that to the pr branch. Currently a merge conflict is present and that'll hold up the tests/mergability

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [inline code/game/objects/items/armor_polish.dm]: You want strictly lesser than here, not lesser or equal as if you have 30/30/20/25, you will still use up the polish but without having any effects

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [inline code/game/objects/items/armor_polish.dm]: For the love of the god of readability

```suggestion
			I.armor = I.armor.setRating(
				melee = I.armor.melee < 30 ? 30 : I.armor.melee,
				bullet = I.armor.bullet < 30 ? 30 : I.armor.bullet,
				laser = I.armor.laser < 20 ? 20 : I.armor.laser,
				energy = I.armor.energy < 25 ? 25 : I.armor.energy
			)
```

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [inline code/game/objects/items/armor_polish.dm]:
```suggestion
			to_chat(user, "You apply [src] to the [target.name].")
```

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [inline code/game/objects/items/armor_polish.dm]: Plural
```suggestion
				to_chat(user, "<span class='warning'>The [src] has [remaining_uses] use[remaining_uses > 1 ? "s" : ""] left.</span>")
```

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [inline code/game/objects/items/armor_polish.dm]:
```suggestion
		to_chat(user, "<span class='warning'>You can only polish suits and headgear!</span>")
```

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [inline code/game/objects/items/armor_polish.dm]: Calling a helmet a "hat" feels weird

```suggestion
				to_chat(user, "<span class='warning'>This headgear is strong enough already! Try it on something weaker.</span>")
```

File: code/modules/uplink/uplink_items.dm
Comment (alexkar598) [inline code/modules/uplink/uplink_items.dm]: Optional since I don't have any better word to suggest here but if you can think of something to replace "that go over your jumpsuit", that'd be great.

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [inline code/game/objects/items/armor_polish.dm]:
```suggestion
		. += "It has [remaining_uses] use[remaining_uses > 1 ? "s" : ""] left."
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items.dm
Comment (ToasterBiome) [reply code/modules/uplink/uplink_items.dm]: should "over your jumpsuit" be just "in your suit slot"? I'm unaware of how else to word it.

File: code/game/objects/items/armor_polish.dm
Comment (alexkar598) [reply code/game/objects/items/armor_polish.dm]: This has not been resolved

File: code/modules/uplink/uplink_items.dm
Comment (alexkar598) [reply code/modules/uplink/uplink_items.dm]: Then leave it as is I suppose

**Review by alexkar598 — CHANGES_REQUESTED** [review]
The if check still needs to be turned into strictly lesser rather than lesser or equal

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**
